components/Animate/Swap.js

```diff
@@ -1,6 +1,6 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import TransitionGroup from 'react-addons-css-transition-group';
+import { TransitionGroup } from 'react-transition-group';
```

Comment:
We should remove the `react-addons-css-transition-group` decency from our package.json